pkg/proxy/iptables/proxier.go

```diff
@@ -1100,17 +1096,18 @@ func (proxier *Proxier) syncProxyRules() {
 			// machine, hold the local port open so no other process can open it
 			// (because the socket might open but it would never work).
 			if (svcInfo.Protocol() != v1.ProtocolSCTP) && localAddrSet.Has(net.ParseIP(externalIP)) {
```

Comment:
just skip v1.ProtocolUDP too to not hold the local port, I want to discard that is this who is swallowing the packet

Reply:
sorry wong place, this is only for externalIPs